src/practices/LanguageIndependent/TimeToSolvePractice.ts

```diff
@@ -0,0 +1,55 @@
+import moment from 'moment';
+import { PracticeContext } from '../../contexts/practice/PracticeContext';
+import { PracticeEvaluationResult, PracticeImpact } from '../../model';
+import { GitServiceUtils } from '../../services/git/GitServiceUtils';
+import { DxPractice } from '../DxPracticeDecorator';
+import { IPractice } from '../IPractice';
+
+@DxPractice({
+  id: 'LanguageIndependent.TimeToSolve',
+  name: '',
+  impact: PracticeImpact.medium,
+  suggestion: '',
+  reportOnlyOnce: true,
+  url: 'https://dxkb.io/p/pull-requests',
+  dependsOn: { practicing: ['LanguageIndependent.DoesPullRequests'] },
+})
+export class TimeToSolvePractice implements IPractice {
+  async isApplicable(): Promise<boolean> {
+    return true;
+  }
+
+  async evaluate(ctx: PracticeContext): Promise<PracticeEvaluationResult> {
+    if (ctx.fileInspector === undefined || ctx.collaborationInspector === undefined) {
+      return PracticeEvaluationResult.unknown;
+    }
+
+    const repoName = GitServiceUtils.getRepoName(ctx.projectComponent.repositoryPath, ctx.projectComponent.path);
+    const ownerAndRepoName = GitServiceUtils.getOwnerAndRepoName(repoName);
+
+    //Both GitHub API and Bitbucket API returns open pullrequests defaultly
+    const pullRequests = await ctx.collaborationInspector.getPullRequests(ownerAndRepoName.owner, ownerAndRepoName.repoName);
+
+    if (pullRequests.items.length === 0) {
+      return PracticeEvaluationResult.practicing;
+    }
```

Comment:
It's not true. If the developer doesn't have PRs, we cannot mark this practice as practicing. This Practice should depend on `DoesPRsPractice`